### PR TITLE
single msg - optional body field

### PR DESCRIPTION
**Pull request by Heleen-cr**

### CODE REVIEW

**Comment by gusliedke** [comment]
Can you create an example in `md` file? Also add a snapshot test for this variation.

File: src/components/Molecules/SingleMessage/SingleMessage.test.js [hunk 1]
```diff
@@ -68,6 +68,21 @@ it('renders Single Message with full width correctly', () => {
...
+it('renders Single Message with full width image and no text correctly', () => {
+  const tree = renderWithTheme(
+    <SingleMessage
+      backgroundColor="purple"
+      imageSet={data.images}
+      image={data.image}
+      imageAltText=""
+      copyFirst={false}
+      fullImage
+    ></SingleMessage>
```
Comment (LouiaR) [inline src/components/Molecules/SingleMessage/SingleMessage.test.js]: I think we don't need to add `</SingleMessage>`

File: src/components/Molecules/SingleMessage/SingleMessage.md [hunk 2]
```diff
@@ -52,6 +52,24 @@ import Link from '../../Atoms/Link/Link';
...
+
+<SingleMessage
+  backgroundColor="purple_dark"
+  imageSet={data.images}
+  image={data.image}
+  imageAltText=""
+  copyFirst={false}
+  fullImage={true}
+  vhFull
+></SingleMessage>;
```
Comment (LouiaR) [inline src/components/Molecules/SingleMessage/SingleMessage.md]: I think we don't need to add </SingleMessage>

File: src/components/Molecules/SingleMessage/SingleMessage.test.js (hunk 1 above)
Comment (Heleen-cr) [reply src/components/Molecules/SingleMessage/SingleMessage.test.js]: indeed we don't

**Pull request merged**